### PR TITLE
RFR: simplified overview section

### PR DESCRIPTION
**Pull request by LindsayHill**
I've taken a lot of stuff out of the Overview - mainly the stuff about IP Fabrics in general. Users can read the BVD & IP Fabric Config  Guide to see how that works.

What do you think @humblearner ?

### CODE REVIEW

File: docs/source/solutions/ipfabric/overview.rst [hunk 1]
```diff
@@ -5,22 +5,23 @@ IP Fabric Solution
...
-    #
-    MORE INFO HERE
-    #
-
-
-|Ipf| can provision and validate Brocade IP Fabrics and BGP-EVPN with minimal effort,
-while support for open interfaces and commonly-used infrastructure programming tools
-enables simple and straightforward customization when needed.
+The |ipf| is an add-on package for |bwc| that provides services and pre-built automations for managing
+Brocade IP Fabrics. It can provision and validate Brocade IP Fabrics and BGP-VPN with minimal effort. It can
```
Comment (humblearner) [inline docs/source/solutions/ipfabric/overview.rst]: s/VPN/EVPN

File: docs/source/solutions/ipfabric/overview.rst [hunk 2]
```diff
@@ -29,112 +30,24 @@ Through the |ipf| CLI, network engineers are able to perform the following tasks
...
-the current configuration on the switches, and displays IP Fabric
-topology in a selected format (such as a PDF, JPEG, or PNG).
-
-.. figure:: ../../_static/images/ipfabric_topology.jpg
-    :align: center
-
-    **IP fabric topology**
-
------------------
-
```
Comment (humblearner) [inline docs/source/solutions/ipfabric/overview.rst]: I think we should leave this part in the doc. as it is about ipf solution.

File: docs/source/solutions/ipfabric/overview.rst (hunk 1 above)
Comment (LindsayHill) [inline docs/source/solutions/ipfabric/overview.rst]: Good catch, pushed fix

File: docs/source/solutions/ipfabric/overview.rst (hunk 2 above)
Comment (LindsayHill) [inline docs/source/solutions/ipfabric/overview.rst]: I'll see about writing a shorter version, and push that

**Comment by humblearner** [comment]
LGTM

**Pull request merged**